backend/requirements.txt

```
@@ -11,3 +11,4 @@ numpy>=2.2.6,<3
 scipy>=1.15.3,<2
 PyYAML>=6.0.2,<7
 orjson>=3.10.18,<4
+valkey >=6.1.0
```

```suggestion
valkey >=6.1.0,<7
```

Making sure that it doesn't break automatically if v7 is released. also consistently with the other requirements

---

Any update could be breaking why not just have it not go up to 6.1.1